.neuro/live.yml

```diff
@@ -0,0 +1,106 @@
+kind: live
+
+defaults:
+  preset: gpu-k80-small-p
+  life_span: 1d
+
+volumes:
+  data:
+    remote: storage:$[[ flow.project_id ]]/data
+    mount: /project/data
+    local: data
+  code:
+    remote: storage:$[[ flow.project_id ]]/code
+    mount: /project/code
+    local: code
+  config:
+    remote: storage:$[[ flow.project_id ]]/config
+    mount: /project/config
+    local: config
+    read_only: True
+  notebooks:
+    remote: storage:$[[ flow.project_id ]]/notebooks
+    mount: /project/notebooks
+    local: notebooks
+  results:
+    remote: storage:$[[ flow.project_id ]]/results
+    mount: /project/results
+    local: results
+  project:
+    remote: storage:$[[ flow.project_id ]]
+    mount: /project
+    local: .
+
+images:
+  myimage:
+    ref: image:$[[ flow.project_id ]]
+    dockerfile: $[[ flow.workspace ]]/Dockerfile
+    context: $[[ flow.workspace ]]/
+    build_preset: cpu-small
+
+jobs:
```

Comment:
We also define a local hypertrain action as a live mode job.
```
jobs:
   ...
   hypertrain:
      action: ws:.neuro/hypertrain.yml
      args:
       ...
```
wdyt?

Reply:
In what case that would be used?

Reply:
Only for cases, when someone wants to run this action from the live mode only, I'm afraid. So yeah, there is almost no moto. 🙂

Later we could go further and publish the batch pipeline as an action in our `neuro-actions`.
But yes, referring to [this](https://github.com/neuro-inc/ml-recipe-hyperparam-wandb/pull/3#discussion_r560961073), it might not be the priority. Let's leave it as is.

Reply:
It required some changes to support the new version of wandb, but it's finally ready :)